src/Carousel.js

```diff
@@ -269,6 +288,7 @@ class Carousel extends React.Component {
           onClick={this.handlePrev}
         >
           {prevIcon}
+          {prevLabelNode}
```

Comment:
looks good, could you move the `prevLabel && <span className="sr-only">` inline here, ditto for the nextLabel as well